src/main/java/com/uid2/shared/attest/UidCoreClient.java

```diff
@@ -286,4 +296,14 @@ public void setPreSignedUrlExpiry(long expiry) {
     public String mask(String cloudPath) {
         throw new UnsupportedOperationException("UidCoreClient::mask method is not supported");
     }
+
+    public void setResponseStatusWatcher(Handler<Integer> watcher) {
```

Comment:
Should the watcher be specified in constructor? Is there a need for a setter?

Reply:
In operator, UidCoreClient is created first, then shutdown handler is created during route setup of operator vertice.

Reply:
In operator, UidCoreClient is created first, then shutdown handler is created during route setup of operator vertice.

Reply:
This does not have to be the case though.

Reply:
I'd prefer to create shutdown handler inside `UIDOperatorVerticle` instead of creating it beforehand just for passing to UidCoreClient ctor.